README.md

```diff
@@ -1 +1,3 @@
-# credentials-context
\ No newline at end of file
+# credentials-context
+
+See: https://github.com/w3c/vc-data-model
```

Comment:
You could slurp in and adjust the did-context README: https://github.com/digitalbazaar/did-context/blob/master/README.md

Reply:
@davidlehn i'll add an issue for improving the readme with your suggestion.  I don't want this to hold up a 1.0 release.